web_export_view/static/src/xml/web_export_view_template.xml

```diff
@@ -2,11 +2,8 @@
 
 <templates id="template" xml:space="preserve">
 <t t-name="AddExportViewMain">
-    <div class="oe_form_dropdown_section">
-        <button class="oe_dropdown_toggle oe_dropdown_arrow">Export Current View</button>
-        <ul class="oe_dropdown_menu">
-            <li class="oe_sidebar_export_view_xls"><span>Excel</span></li>
-        </ul>
+    <div class="o_cp_sidebar btn-group btn-group-sm">
+        <button class="btn btn-default export_treeview_xls" type="button"><i class="fa fa-file-excel-o"></i></button>
```

Comment:
It would be nice if this were on multiple lines, the wrapping caused me to have to double take that it was all the same line.

Also, that `<i>` should have a `t-translation="off"` in order to avoid funky translations